wiki/Beatmap/en.md

```diff
@@ -1,38 +1,52 @@
-# Beatmaps
+---
+tags:
+  - mapset
+  - beatmapset
+---
 
-*Not to be confused with [Difficulties](/wiki/Difficulties).*
-
-**Beatmaps** are game levels created for songs and stored in text files with [the `.osu` extension](/wiki/osu!_File_Formats/Osu_(file_format)). They have different structure and sometimes may be played only in one [game mode](/wiki/Game_mode).
+# Beatmap
 
 ## Overview
 
-At the core, a beatmap consists of [hit objects](/wiki/Hit_object) of a certain [game mode](/wiki/Game_mode), which are set up accordingly to the [timing](/wiki/Glossary#timing-section) tailored to a specific song. Beatmaps have different aspects, which directly affect user experience.
+A **beatmap** (sometimes called *beatmapset*) is a set of game levels, or [difficulties](#difficulty), that are composed of various [hit objects](/wiki/Hit_object) and represent a single song. It also includes other components, all packed in an archive with [the `.osz` extension](/wiki/osu!_File_Formats/Osz_(file_format)):
```

Comment:
afaik you can use more than one song in a beatmapset (e.g., https://osu.ppy.sh/beatmapsets/72474#osu/207087) although idk the technical details of that

Reply:
it's possible, but at least forbidden by RC. I don't know the official status of this, but may add a very small note instead (like a single word)